package.json

```diff
@@ -27,19 +27,17 @@
   },
   "license": "Apache-2.0",
   "dependencies": {
-    "bluebird": "^3.5.0",
-    "debug": "^3.0.1",
-    "kuzzle-sdk": "^5.0.7",
-    "lodash": "^4.17.4"
+    "debug": "^3.1.0",
+    "kuzzle-sdk": "^5.0.11",
+    "lodash": "^4.17.10"
   },
   "devDependencies": {
-    "eslint": "^2.9.0",
-    "istanbul": "^0.4.3",
-    "kuzzle-common-objects": "^3.0.4",
-    "mocha": "^2.4.5",
-    "rewire": "^2.5.2",
-    "should": "^8.3.1",
-    "sinon": "^1.17.4"
+    "bluebird": "^3.5.1",
```

Comment:
Why is bluebird in the development dependencies?

Reply:
why not just returning a native promise then?

Reply:
Of course we can !!
Why did I used Bluebird ? :/

Reply:
fixed ;)